### ISSUE TITLE
spec-cleaner mangles kernel-firmware

### ISSUE THREAD

**Issue by plusky** [openSUSE/spec-cleaner#171]
```
# spec-cleaner -i kernel-firmware.spec
# osc diff
Index: kernel-firmware.spec
===================================================================
--- kernel-firmware.spec	(revision 686f8caa4b945b034b384e2fc87dd18c)
+++ kernel-firmware.spec	(working copy)
@@ -20,7 +20,6 @@
 %define __ksyms_supplements bash %{_sourcedir}/firmware.sh --cache %{_builddir}/%{?buildsubdir}/find_supplements.cache --find-supplements
 %define __ksyms_path ^/lib/firmware
 %define version_unconverted 20170217
-
 Name:           kernel-firmware
 Version:        20170217
 Release:        0
@@ -74,64 +73,4 @@
 previously included in the kernel. It is shared by all kernels >=
 2.6.27-rc1.
 
-%package -n ucode-amd
-Summary:        Microcode updates for AMD CPUs
-Group:          System/Kernel
-Requires(post): coreutils
-Requires(postun): coreutils
-Supplements:    modalias(x86cpu:vendor%%3A0002%%3Afamily%%3A*%%3Amodel%%3A*%%3Afeature%%3A*)
-
-%description -n ucode-amd
-This package contains the microcode files used by AMD CPUs.
-
-%prep
-%setup -q
-cp %{SOURCE2} .
-
-%build
-# nothing to do
-
-%install
-mkdir -p %{buildroot}/lib/firmware
-cp -avf * %{buildroot}/lib/firmware
-rm -f %{buildroot}/lib/firmware/WHENCE
-bash %{_sourcedir}/firmware.sh --kill-duplicates %{buildroot}/lib/firmware/
-%fdupes %{buildroot}
-rm -f find_supplements.cache
-# In alsa-firmware
-rm -f %{buildroot}/lib/firmware/ctefx.bin
-rm -f %{buildroot}/lib/firmware/ctspeq.bin
-# Devel files
-rm -rf %{buildroot}/lib/firmware/carl9170fw
-
-%post
-%{?regenerate_initrd_post}
-
-%post -n ucode-amd
-%{?regenerate_initrd_post}
-
-%postun
-%{?regenerate_initrd_post}
-
-%postun -n ucode-amd
-%{?regenerate_initrd_post}
-
-%posttrans
-%{?regenerate_initrd_posttrans}
-
-%posttrans -n ucode-amd
-%{?regenerate_initrd_posttrans}
-
-%files
-%defattr(0644,root,root,0755)
-%doc WHENCE README GPL-2 GPL-3 LICEN[CS]E.*
-%exclude /lib/firmware/amd-ucode
-%exclude /lib/firmware/LICENSE.amd-ucode
-/lib/firmware/*
-
-%files -n ucode-amd
-%defattr(0644,root,root,0755)
-/lib/firmware/amd-ucode
-/lib/firmware/LICENSE.amd-ucode
-
 %changelog
```

**Comment by scarabeusiv** [comment]
It is caused by:

Supplements:    modalias(x86cpu:vendor%%3A0002%%3Afamily%%3A*%%3Amodel%%3A*%%3Afeature%%3A*)


I checked it further and it seems to have problems in parsing those %% sections, if you delete those everything proceeds as anticipated...

**Comment by scarabeusiv** [comment]
This was fixed in the repository by commit baf06ff3aeea0dc5e79533f59d2bd873d6210f93
